rpcs3/rpcs3qt/localized.cpp

```diff
@@ -8,102 +8,52 @@ QString Localized::GetVerboseTimeByMs(qint64 elapsed_ms, bool show_days) const
 {
 	if (elapsed_ms <= 0)
 	{
-		return "";
+		return tr("Never played");
 	}
 
 	const qint64 elapsed_seconds = (elapsed_ms / 1000) + ((elapsed_ms % 1000) > 0 ? 1 : 0);
-
 	qint64 hours = elapsed_seconds / 3600;
 
-	// For anyone who was wondering why there need to be so many cases:
-	// 1. Using variables won't work for future localization due to varying sentence structure in different languages.
-	// 2. The provided Qt functionality only works if localization is already enabled
-	// 3. The provided Qt functionality only works for single variables
-
-	if (show_days && hours >= 24)
-	{
-		const qint64 days  = hours / 24;
+	const qint64 days = hours / 24;
```

Comment:
This entire function is already existing as a draft PR. I don't understand why you're trying to do this again. Especially when it was made clear that these types of translations won't get merged until we ship English translation files